client/clientmanager/src/main/java/io/mosip/registration/clientmanager/dto/registration/RegistrationDto.java

```diff
@@ -205,25 +204,41 @@ public void addBiometric(String fieldId, String attribute, int attempt, Biometri
 
     public List<BiometricsDto> getBiometrics(String fieldId, Modality modality, int attempt) {
         List<BiometricsDto> list = new ArrayList<>();
-        for(String attribute : modality.getAttributes()) {
+        for (String attribute : modality.getAttributes()) {
             String key = String.format(BIO_KEY_ATTEMPT, fieldId, attribute, attempt);
-            if(this.biometrics.containsKey(key))
+            if (this.biometrics.containsKey(key))
                 list.add(this.biometrics.get(key));
         }
         return list;
     }
 
+    public BiometricsDto getBestBiometrics(String fieldId, String attribute) {
```

Comment:
Need to remove this duplicate method and directly call the existing getBestBiometrics(String fieldId, Modality modality) in the PreviewActivity.